client/index.js

```diff
@@ -1,2 +1,16 @@
-import './view/base/index.js'
-import './view/browsers/index.js'
+import { initForm } from './view/Form/form.js';
+import { initQueryLinks } from './view/QueryLink/queryLink.js';
+
+initForm();
+
+initQueryLinks();
+
+//TODO change url when change query and get query from url when page are loading
+
+// document.addEventListener('DOMContentLoaded', () => {
```

Comment:
BTW, we don’t need `document.addEventListener('DOMContentLoaded'` since `<script>` has `defer`